packages/libsql-client/src/lib/driver/HttpDriver.ts

```diff
@@ -26,7 +26,7 @@ export class HttpDriver implements Driver {
 
         const statements = buildStatements(["BEGIN", ...stmts, "COMMIT"]);
 
-        const response = await fetch(this.url, {
+        const response = await fetch(this.url + "/queries", {
```

Comment:
I would suggest using `new URL("/queries", this.url)` for more robust URL joining.